include/battle.h

```diff
@@ -519,6 +519,11 @@ struct LostItem
     u16 stolen:1;
 };
 
+struct BattleRulesVariant 
+{
+    bool8 skyBattle:1;
```

Comment:
Any reason this is in a struct? Not saying it has to be changed, just curious what the reasoning was.

Reply:
I didn't write the original implementation, when I got here, it was already a struct. Happy to change it to something else.

Reply:
I would personally get rid of the struct and just put 
u8 isSkyBattle:1;
in the BattleStruct